From 5083232273c93f9115bf91bed373d8ab749106d9 Mon Sep 17 00:00:00 2001
From: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
Date: Fri, 17 Dec 2021 15:33:55 +0300
Subject: [PATCH 084/120] zdtm: remove mntns-deleted-dst test leftover from git

Looks like in commit [1] we've non-intentionally added this tmp file to
git, let's remove it.

Fixes: 01ee29702 ("s390:zdtm: Enable zdtm for s390") [1]
Signed-off-by: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
---
 test/zdtm/static/mntns-deleted-dst | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 delete mode 100644 test/zdtm/static/mntns-deleted-dst

diff --git a/test/zdtm/static/mntns-deleted-dst b/test/zdtm/static/mntns-deleted-dst
deleted file mode 100644
index e69de29bb..000000000
-- 
2.34.1

